src/components/input/demo/index.tsx

```diff
@@ -330,6 +334,16 @@ export class SmoothlyInputDemo {
 				<smoothly-item value="12">December</smoothly-item>
 			</smoothly-input-select>,
 			<button onClick={async () => this.selectElement.reset()}>Reset select</button>,
+
+			<smoothly-form looks="border" onSmoothlyFormSubmit={e => console.log("Submitted", e.detail)}>
+				<smoothly-input-file placeholder="Select or drag a file here" name="input file demo">
+					{" "}
```

Comment:
This is safe to remove. it gets added by the auto formatter when it thinks a trailing space is important and it breaks it up to multi line.